src/App.js

```diff
@@ -1,7 +1,8 @@
 import React, { Component } from 'react';
 import { Route, Link } from 'react-router-dom'
 import './css/index.css';
-import { headerNav } from './header.js'
+import { headerNav } from './components/headerNav'
```

Comment:
What is the difference between `import { headerNav } from './components/headerNav'` and `import headerNav from './components/headerNav'` would the code still work if you use `import headerNav from './components/headerNav'`?

Reply:
removed { } , used only for multiple objects from import